libbeat/scripts/Makefile

```diff
@@ -209,7 +209,8 @@ test: unit
 
 .PHONY: testsuite
 testsuite: ## @testing Runs all tests and generates the coverage reports
-testsuite: clean update
```

Comment:
`clean` and `update` won't run in parallel, right now from what I can tell (on my laptop) they end up running in parallel and `update` tries to invoke `activate` from the `build` dir right after `clean` deleted all of it